docs/resources/driver.md

```diff
@@ -0,0 +1,217 @@
+---
```

Comment:
In index.md, we are saying that rafay_driver is in development? then why are we adding docs for it?

Reply:
yeah, it is scheduled for next release hence added it under in development. will remove it now and can be added later